### PR TITLE
Cursed Dice 2; Electric Boogaloo

### PR DESCRIPTION
**Pull request by Arthurdentist**
<h1>I fucked up my git whilst attempting to squash commits edition</h1>

old PR: # #7406

### CODE REVIEW

File: code/game/objects/items/weapons/dice.dm [hunk 1]
```diff
@@ -122,3 +122,172 @@
...
+/obj/item/weapon/dice/d20/cursed
+	desc = "Something about this dice seems wrong"
+	name = "/improper Mysterious d20"
+	var/deactivated = 0 //Eventually the dice runs out of power
+
+/obj/item/weapon/dice/d20/cursed/pickup(mob/user as mob)
+	if(deactivated == 0)
+		to_chat(user, "<span class=sinister><B>Are you feeling lucky?</span></B>")
+	else
+		return 0
```
Comment (clusterfack) [inline code/game/objects/items/weapons/dice.dm]: What is this for

File: code/game/objects/items/weapons/dice.dm [hunk 2]
```diff
@@ -122,3 +122,172 @@
...
+		log_game("[key_name(user)] has [thrown? "used" : "thrown"] a cursed dice and rolled a [result]")
+		if(istype(user,/mob/living/carbon/human)) //check that a humanoid is rolling the dice; Xenomorphs / Sillicons need not apply.
+			var/mob/living/carbon/human/h = user
+			switch(result)
+				if(1)
+					to_chat(user, "<span class=sinister><B>A natural failure, your poor roll has cursed you. Better luck next time! </span></B>")
+					flick("e_flash", user.flash)
+					h.Cluwneize()
+				if(2 to 4)
+					to_chat(user, "<span class=sinister><B>It could be worse, but not much worse! Enjoy your curse! </span></B>")
```
Comment (clusterfack) [inline code/game/objects/items/weapons/dice.dm]: Actually rhyming worse with worse, what a world

File: code/game/objects/items/weapons/dice.dm [hunk 3]
```diff
@@ -122,3 +122,172 @@
...
+							h.adjustBrainLoss(200)
+							user.reagents.add_reagent("nutriment", 1000)
+							user.overeatduration = 1000
+
+							to_chat(user, "<span class=danger><B>In this moment you feel euphoric! </span></B>")
+				if(10 to 12)
+					to_chat(user, "<span class=sinister><B>You get nothing. No curse or reward! </span></B>")
+				if(13)
+					to_chat(user, "<span class=sinister><B>You've rolled 13! The cursed dice is broken! </span></B>")
+					explosion(get_turf(src), 0, 0, 4, 7)
```
Comment (clusterfack) [inline code/game/objects/items/weapons/dice.dm]: I hope you realize the flash range does nothing

File: code/game/objects/items/weapons/dice.dm (hunk 1 above)
Comment (Arthurdentist) [inline code/game/objects/items/weapons/dice.dm]: else return 0 could be removed which I intend to do, just forgot last night.

The dice deactivates itself after a few rolls and becomes essentially a normal dice (though a badmin can re-activate it through variable editing).

File: code/game/objects/items/weapons/dice.dm (hunk 2 above)
Comment (Arthurdentist) [inline code/game/objects/items/weapons/dice.dm:149]: not an intentional rhyme

File: code/game/objects/items/weapons/dice.dm (hunk 3 above)
Comment (Exxion) [inline code/game/objects/items/weapons/dice.dm:227]: Wait, why not? Is it intended that it doesn't work?

Comment (clusterfack) [inline code/game/objects/items/weapons/dice.dm]: It was a side project of one of the coders about a year and a half ago, either dylan or nexis. It had some bugs and was completely dropped. Anyone is welcome to finish the system if they want.

**Pull request merged**